modules/core/category.py

```diff
@@ -22,7 +22,7 @@ def category(self):
         logger.separator("Updating Categories", space=False, border=False)
         torrent_list = self.qbt.get_torrents({"category": "", "status_filter": "completed"})
         for torrent in torrent_list:
-            new_cat = self.qbt.get_category(torrent.save_path)
+            new_cat = self.get_tracker_cat(torrent) or self.qbt.get_category(torrent.save_path)
```

Comment:
Can we swap it so the original save_path takes priority?
```suggestion
            new_cat = self.qbt.get_category(torrent.save_path) or self.get_tracker_cat(torrent)
```

Reply:
The problem with this is that `self.qbt.get_category(torrent.save_path)` never returns `None` or an empty value, so `get_tracker_cat` is never called. And for me, it seems more logical to give precedence to the category defined in the tracker section because it is optional.

If you really want to swap it, a solution would be to merge `get_tracker_cat` and `get_category` so that save_path takes precedence, but if a there's no category matching a save_path, we use the category defined in the trackers option if applicable, instead of creating a default category.